docs/products/kafka/howto/best-practices.md

```diff
@@ -2,86 +2,111 @@
-You can specify a value for acknowledgements setting `acks` in the
-client producer configuration. This will have an impact on how the
-success of a write operation is determined.
+The `acks` parameter in the producer configuration controls how the success of a
+write operation is determined. Choose the appropriate setting based on your data
+reliability needs:
 
-With `acks` equal to **0** after the producer sends the data, it does
-not wait for a confirmation from the broker. This will make
-communication faster. However, there is a potential loss of data in case
-of the broker being down when the producer sends the data. This
-configuration is only appropriate when you can afford loss of data.
+- **`acks=0`**: The producer sends data without waiting for confirmation from the
+  broker. This speeds up communication, but there’s a risk of data loss if the broker
+  goes down during transmission. Use this setting only if some data loss is acceptable.
 
-With `acks` equal to **1** (default value and recommended behaviour),
-the producer waits for the leader broker to acknowledge that the data
-was received. This mode partially prevents data loss, however, the data
-loss still can occur if the broker goes down between the moment it sent
-acknowledgement and the data was replicated.
+- **`acks=1` (default and recommended setting)**: The producer waits for the leader
+  broker to confirm receipt of the data. This reduces the chance of data loss, but
+  data can still be lost if the leader fails before the data is fully replicated.
 
-With `acks` equal to **all**, the leader and all the replicas will send
-confirmation of the received data. This configuration slows the
-communication, but ensures that there will be no data loss, since the
-replicas also confirm that the data was received.
+- **`acks=all`**: The producer waits for acknowledgment from both the leader and all
+  replicas. This ensures no data loss but can slow down communication.
+
+## Configure single availability zone (AZ) for BYOC customers
+
+For Bring Your Own Cloud (BYOC) customers, deploying Aiven for Apache Kafka in a single
+availability zone (AZ) can reduce costs by removing inter-zone data transfer fees.
+However, using a single AZ removes Kafka's resiliency, as data is not replicated across
+zones. This increases the risk of downtime if the AZ fails.
+
+When considering a single AZ allocation, evaluate your organization's risk tolerance,
+as Aiven's standard uptime SLA does not apply to services deployed in a single AZ.
+
+- To enable this option for your project, contact
+  [Aiven support](mailto:support@aiven.io) or your account team.
+- You must configure single AZ allocation during service creation. It cannot be applied
+  to existing services.
+
+To enable single AZ allocation, use the [Aiven CLI](/docs/tools/cli) and
+set `single_zone.enabled=true`.
+
+Example command:
+
+```bash
+avn service create SERVICE_NAME           \
+  --service-type SERVICE_TYPE             \
```

Comment:
For this example can we put Kafka as the service type? I realize it might be a command that can be used for other services, but if this example is for Kafka it might be nice to fill that part in?